src/libraries/System.Private.CoreLib/src/System/Index.cs

```diff
@@ -140,11 +141,18 @@ public override string ToString()
 
         private string ToStringFromEnd()
         {
+#if SYSTEM_PRIVATE_CORELIB || NETCOREAPP
```

Comment:
Is netcoreapp not set when building corelib?

Reply:
When I first wrote this I didn't think it was. But I just checked and it is. Also worth noting - so is `NETCOREAPP2_1` because of this:

https://github.com/dotnet/runtime/blob/54e3d38e4948fe3c285cf1a7bda0277f3b28e0df/src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj#L9

I've changed all the `#if`s in the latest code since I needed to add `netstandard2.1`. There are no more `#if SYSTEM_PRIVATE_CORELIB || NETCOREAPP`.